### ISSUE TITLE
support for repairing all keyspaces

### ISSUE THREAD

**Issue by jaibheem** [thelastpickle/cassandra-reaper#24]
Hello,

Currently when scheduling or running a repair there is a filed for keyspace, but is it possible to have a wild card * so that all the keyspaces will be repaired.

In my usecase we have multiple keyspaces that gets generated based on the req, it is very much impossible to use keyspace name to repair

**Comment by zznate** [comment]
Understood - but for most clusters, it would be a very bad idea to launch multiple repairs against multiple keyspaces simultaneously.

Retrieving the name of a keyspace should be straight forward enough.

**Comment by TheTomahawk** [comment]
I would second this request.  However, I'd be happy that it don't repair all keyspaces simulteneously, that it wait until keyspace x is completed until moving on to keyspace y.  Once all are done, it just starts again.  (i.e. keyspaces in sequences, but repairs for the current keyspace in parallel).

This way I don't need to care about what keyspaces are on any of the clusters, I can just point an instance of Reaper against a cluster and it will repair everything constantly - setup and forget (and monitor!), leaving it to run completely automated.

**Comment by patrickclee0207** [comment]
the auto schedule feature would be nice, like this fork https://github.com/eBayClassifiedsGroup/cassandra-reaper, it would automatically schedule the keyspaces for you on startup.  i've tried it and worked pretty good other than the exclude list is hardcoded in the code.

**Comment by adejanovski** [comment]
Thanks for the pointer @patrickclee0207 , I've pulled and rebased the PR that was supposed to bring that feature into the Spotify repo by @sebbonnet : https://github.com/spotify/cassandra-reaper/pull/143

It's available in branch https://github.com/thelastpickle/cassandra-reaper/tree/autoschedule and I opened PR #55 

Feel free to test and send feedback, thanks !

**Comment by patrickclee0207** [comment]
nice!! i've pulled it down and deployed it to for a couple clusters.  works really nice so far.  the configurable exclude list is great!

**Comment by adejanovski** [comment]
Merged into master
